src/bundle/Controller/ContentTypeController.php

```diff
@@ -148,14 +169,42 @@ public function addAction(ContentTypeGroup $group): Response
         ]);
     }
 
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup $group
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     */
     public function editAction(ContentTypeGroup $group, ContentType $contentType): Response
     {
+        // It is possible to have two user wants to edit the same Content Type. Due the Kernel it is not possible.
```

Comment:
How about simplifying this line:
`// Kernel does not allow editing the same Content Type simultaneously by more than one user`
the next line is fine.